### PR TITLE
{bio}[foss/2022a] vConTACT2 v0.11.3 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3127.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/b9459a176e15e0874c61f6da777d3ccc for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 22.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.10.6
See https://gist.github.com/71f0fcb7cb5fdb6bdb2f740cac232b59 for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
pg-interactive.hpc.rug.nl - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/4e5c8f156955f820a1dc08a954d38977 for a full test report.

File: easybuild/easyconfigs/v/vConTACT2/vConTACT2-0.11.3-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        'checksums': ['af097bae1b616dde5c5744441e2ddc69e74dfdcb0c263129610d85b87445a59d'],
+    }),
+    ('tzlocal', '4.2', {
+        'checksums': ['ee5842fa3a795f023514ac2d801c4a81d1743bbe642e3940143326b3a00addd7'],
+    }),
+    ('clusterone', '0.15.3', {
+        'checksums': ['ce253e778c68aa1c8f408ac2b835caca62d90b9c9206bf9f56819ee97b198cf9'],
+        # remove strict version requirements for required Python packages,
+        # remove requirement for enum34 (not needed when using Python >= 3.4)
+        'preinstallopts': "sed -i 's/==/>=/g' setup.py && sed -i '/enum34/d' setup.py && ",
```
Comment (bedroge) [inline easybuild/easyconfigs/v/vConTACT2/vConTACT2-0.11.3-foss-2022a.eb:67]: @boegel I was just checking if it's possible to add some sanity check commands, and then ran into this:
```
$ just -h
Traceback (most recent call last):
  File "/data/apps/software/vConTACT2/0.11.3-foss-2022a/bin/just", line 5, in <module>
    from clusterone.clusterone_cli import main
  File "/data/apps/software/vConTACT2/0.11.3-foss-2022a/lib/python3.10/site-packages/clusterone/clusterone_cli.py", line 81, in <module>
    @click_log.init(__name__)
AttributeError: module 'click_log' has no attribute 'init'
```
So the requirements may have to be somewhat stricter anyway, I guess...

File: easybuild/easyconfigs/v/vConTACT2/vConTACT2-0.11.3-foss-2022a.eb [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    }),
+    (name, version, {
+        'source_urls': ['https://bitbucket.org/MAVERICLab/vcontact2/get/'],
+        'sources': [{'download_filename': '%s.tar.gz' % local_commit, 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['22890d2839d53140220bff8e0333f1dab562c16270cd86dfe88ff0836d8ba0b7'],
+    }),
+]
+
+sanity_pip_check = True
```
Comment (bedroge) [inline easybuild/easyconfigs/v/vConTACT2/vConTACT2-0.11.3-foss-2022a.eb:76]:
```suggestion
sanity_check_commands = [
    'coreapi  --help',
    'just --help',
    'raven --help',
    'vcontact2 --help',
    'vcontact2_gene2genome --help',
]

sanity_pip_check = True
```

Comment (boegel) [reply easybuild/easyconfigs/v/vConTACT2/vConTACT2-0.11.3-foss-2022a.eb:86]: fixed in 3ff5fcdda7

File: easybuild/easyconfigs/v/vConTACT2/vConTACT2-0.11.3-foss-2022a.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/v/vConTACT2/vConTACT2-0.11.3-foss-2022a.eb:68]: Nice catch!

Fixed in 3ff5fcdda7

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3125.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/33cf1dab16dbd3d2205f71c8e616104f for a full test report.

**Comment by bedroge** [comment]
@boegelbot please test @ generoso

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 22.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.10.6
See https://gist.github.com/0691a95444e76ee74adf8b66231c95a3 for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
pg-interactive.hpc.rug.nl - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/8621327eb266584812e78eba79701002 for a full test report.

**Comment by bedroge** [comment]
```
== 2023-02-24 10:29:58,434 run.py:234 INFO running cmd: just --help 
== 2023-02-24 10:30:01,670 easyblock.py:3450 WARNING Sanity check: sanity check command just --help exited with code 1 (output: Traceback (most recent call last):
  File "/data/f115261/easybuildinstall/software/vConTACT2/0.11.3-foss-2022a/bin/just", line 5, in <module>
    from clusterone.clusterone_cli import main
  File "/data/f115261/easybuildinstall/software/vConTACT2/0.11.3-foss-2022a/lib/python3.10/site-packages/clusterone/clusterone_cli.py", line 10, in <module>
    from clusterone import commands
  File "/data/f115261/easybuildinstall/software/vConTACT2/0.11.3-foss-2022a/lib/python3.10/site-packages/clusterone/commands/__init__.py", line 6, in <module>
    from . import get
  File "/data/f115261/easybuildinstall/software/vConTACT2/0.11.3-foss-2022a/lib/python3.10/site-packages/clusterone/commands/get/__init__.py", line 3, in <module>
    from . import events
  File "/data/f115261/easybuildinstall/software/vConTACT2/0.11.3-foss-2022a/lib/python3.10/site-packages/clusterone/commands/get/events/__init__.py", line 1, in <module>
    from .cmd import command
  File "/data/f115261/easybuildinstall/software/vConTACT2/0.11.3-foss-2022a/lib/python3.10/site-packages/clusterone/commands/get/events/cmd.py", line 12, in <module>
    LOCAL_TIMEZONE = get_localzone()
  File "/data/f115261/easybuildinstall/software/vConTACT2/0.11.3-foss-2022a/lib/python3.10/site-packages/tzlocal/unix.py", line 203, in get_localzone
    _cache_tz = _get_localzone()
  File "/data/f115261/easybuildinstall/software/vConTACT2/0.11.3-foss-2022a/lib/python3.10/site-packages/tzlocal/unix.py", line 185, in _get_localzone
    utils.assert_tz_offset(tz)
  File "/data/f115261/easybuildinstall/software/vConTACT2/0.11.3-foss-2022a/lib/python3.10/site-packages/tzlocal/utils.py", line 63, in assert_tz_offset
    raise ValueError(msg)
ValueError: Timezone offset does not match system offset: 0 != 3600. Please, check your config files.
)
```

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
pg-interactive.hpc.rug.nl - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/6a76bc34cababc75bf7ac26411007900 for a full test report.

**Comment by bedroge** [comment]
This was indeed some timezone issue with my build container, solved by setting `$TZ`.

**Comment by bedroge** [comment]
@boegelbot please test @ jsc-zen2

**Review by bedroge — APPROVED** [review]
lgtm

**Comment by bedroge** [comment]
Going in, thanks @boegel!

**Pull request merged**